test/system/070-build.bats

```diff
@@ -877,6 +877,44 @@ EOF
     run_podman rmi -f build_test
 }
 
+@test "podman build .dockerignore failure test " {
+    tmpdir=$PODMAN_TMPDIR/build-test
+    subdir=$tmpdir/subdir
+    mkdir -p $subdir
+
+    cat >$tmpdir/.dockerignore <<EOF
+*
+subdir
+!*/sub1*
+EOF
+    cat >$tmpdir/Containerfile <<EOF
+FROM $IMAGE
+COPY ./ ./
+COPY subdir ./
+EOF
+    run_podman 125 build -t build_test $tmpdir
+    is "$output" ".*Error: error building at STEP \"COPY subdir ./\": no items matching glob" ".dockerignore was ignored"
+}
+
+@test "podman build .containerignore and .dockerignore test" {
+    tmpdir=$PODMAN_TMPDIR/build-test
+    mkdir -p $subdir
+    touch $subdir/test1 $subdir/test2
+    cat >$tmpdir/.containerignore <<EOF
+test2*
+EOF
+    cat >$tmpdir/.dockerignore <<EOF
+test1*
+EOF
+    cat >$tmpdir/Containerfile <<EOF
+FROM $IMAGE
+COPY ./ ./test/
+RUN ls /tmp/test/
+EOF
+    run_podman build -t build_test $tmpdir
+    is "$output" ".*test1" "test1   should exists in the final image" "Confirm only .containerignore is used"
```

Comment:
Too many arguments to `is()`: the last one ("Confirm only...") is ignored. Could you pick one or the other between that and "test1 should exist"?